style/ruby/README.md

```diff
@@ -38,6 +38,9 @@ Ruby
 * Use heredocs for multi-line strings.
 * Prefer `protected` over `private` for non-public `attr_reader`s, `attr_writer`s,
   and `attr_accessor`s.
+* Break up method definitions over 80 characters long by
```

Comment:
Ah... yes, those two rules already handle this. Maybe we just add this example to the second rule? 

`If you break up an argument list, keep the arguments on their own lines and closing parenthesis on its own line. [example]`